llvm/tools/llvm-cas/llvm-cas.cpp

```diff
@@ -295,10 +322,10 @@ int dump(ObjectStore &CAS) {
   return 0;
 }
 
-int makeBlob(ObjectStore &CAS, StringRef DataPath) {
+int makeBlob(ObjectStore &CAS, ArrayRef<std::string> DataPath) {
   ExitOnError ExitOnErr("llvm-cas: make-blob: ");
   std::unique_ptr<MemoryBuffer> Buffer =
-      ExitOnErr(openBuffer(DataPath));
+      ExitOnErr(openBuffer(DataPath.front()));
```

Comment:
function signature change is misleading. This function only takes one input, so `DataPath.front()` should happen at the caller side.